doc/varnish-configuration.rst

```diff
@@ -4,7 +4,7 @@ Varnish Configuration
 ---------------------
 
 Below you will find detailed Varnish configuration recommendations for the
-features provided by this library. The examples are tested with Varnish
+features provided by this library. These examples have been tested with Varnish
```

Comment:
Sure, that works with me - thanks!

--
Justin Finkelstein
Technical Director
ilithium limited
http://uk.linkedin.com/in/justinfinkelstein/

On 10 April 2015 at 18:03, David Buchmann <notifications@github.com> wrote:

> In doc/varnish-configuration.rst
> <https://github.com/FriendsOfSymfony/FOSHttpCache/pull/183#discussion_r28161786>
> :
>
> > @@ -4,7 +4,7 @@ Varnish Configuration
> >  ---------------------
> >
> >  Below you will find detailed Varnish configuration recommendations for the
> > -features provided by this library. The examples are tested with Varnish
> > +features provided by this library. These examples have been tested with Varnish
>
> actually its not examples, but reference documentation. lets say:
>
> ... features provided by this library. The configuration is provided for
> Varnish 3 and 4.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FriendsOfSymfony/FOSHttpCache/pull/183/files#r28161786>
> .
>